src/Serilog/Rendering/Padding.cs

```diff
@@ -21,7 +21,7 @@ static class Padding
     /// <summary>
     /// Writes the provided value to the output, applying direction-based padding when <paramref name="alignment"/> is provided.
     /// </summary>
-    public static void Apply(TextWriter output, string value, in Alignment? alignment)
```

Comment:
https://github.com/JosefPihrt/Roslynator/blob/main/docs/analyzers/RCS1242.md
https://github.com/dotnet/runtime/issues/23969
https://github.com/dotnet/corefx/pull/24997

Reply:
Is making `Alignment` readonly a suitable alternative? (The `in` keyword was added here thanks to advice from a competing analyzer, IIRC 🤷 )

Reply:
> Is making Alignment readonly a suitable alternative? 

`Alignment` is already is readonly struct.